austrakka/components/user/__init__.py

```diff
@@ -37,28 +41,32 @@ def user_list(show_disabled: bool, out_format: str):
 @opt_owner_group_roles(required=False)
 @opt_is_austrakka_process(default=False)
 def user_add(
-    user_id: str,
-    org: str,
-    owner_group_roles: List[str],
-    is_austrakka_process: bool,
+        user_id: str,
+        org: str,
+        owner_group_roles: List[str],
+        is_austrakka_process: bool,
 ):
     """Add users in AusTrakka"""
     add_user(user_id, org, owner_group_roles, is_austrakka_process)
 
 
 @user.command('update', hidden=hide_admin_cmds())
-@click.argument('user-id', type=str)
+@click.argument('object-id', type=str)
+@opt_name(help="Display Name", required=False)
+@opt_email_address(required=False)
 @opt_organisation(required=False)
-@opt_owner_group_roles(required=False)
-@opt_is_austrakka_process()
+@opt_group_role(required=False)
```

Comment:
The help text for opt_group_role says "The group and role to remove from the specified user" which is presumably not correct! Is this meant to be owner group roles specifically?

Reply:
For ordinary group-roles, I'd say we should not be setting those here as the syntax becomes confusing and onerous - do you have to supply the full list of the user's current role assignments on the command line? If you don't, do all their roles get stripped away? I'd say avoid the whole issue by having this command update basic user properties and not touch group role assignments at all, and then we have to use `austrakka group role add/remove` to edit roles.